packages/@ourworldindata/core-table/src/CoreTableUtils.test.ts

```diff
@@ -140,6 +143,44 @@ describe(toleranceInterpolation, () => {
             2,
         ])
     })
+
+    it("interpolates values in both directions", () => {
+        const valuesAsc = [
+            1,
+            ErrorValueTypes.MissingValuePlaceholder,
+            ErrorValueTypes.MissingValuePlaceholder,
+            2,
+        ]
+        const timesAsc = [0, 1, 2, 3]
+        const tolerance = 1
+        toleranceInterpolation(valuesAsc, timesAsc, {
+            timeToleranceForwards: tolerance,
+            timeToleranceBackwards: tolerance,
+        })
+        expect(valuesAsc).toEqual([1, 1, 2, 2])
+    })
+
+    it("interpolates values in the backwards direction", () => {
```

Comment:
I would like to see these tests extended a bit: they should both also cover the first/last element, both because it's easiest to have an off-by-one error in there, and also because that's where we can actually observe the correct insertion of `NoValueWithinTolerance` error values.

So, I suggest `valuesAsc` to look like this for both tests:
```ts
[
	ErrorValueTypes.MissingValuePlaceholder,
	1,
	ErrorValueTypes.MissingValuePlaceholder,
	ErrorValueTypes.MissingValuePlaceholder,
	2,
	ErrorValueTypes.MissingValuePlaceholder
]
```

Reply:
Makes sense :)